src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/indication/FoundIndication.java

```diff
@@ -81,14 +79,17 @@ public FoundIndication(Run build, String originalPattern,
      * @param pattern the pattern we used to match.
      * @param matchingFile the path to the file in which we found the match.
      * @param matchingString the String that makes up the match.
+     * @param matchingLine the line number of the found indication
      */
     @JsonCreator
     public FoundIndication(@JsonProperty("pattern") String pattern,
             @JsonProperty("matchingFile") String matchingFile,
-            @JsonProperty("matchingString") String matchingString) {
+            @JsonProperty("matchingString") String matchingString,
+            @JsonProperty("matchingLine") Integer matchingLine) {
```

Comment:
There might be external uses of the old constructor signature (though unlikely). To be sure we usually leave the old signature around but deprecated.

Reply:
Ok. Brought it back, but marked it @Deprecated

Reply:
I accidentally pinged the user "deprecated" a few times due to the `@`-sign :) Sorry about that, deprecated.

Reply:
Ping @TWestling

Reply:
@rsandell Yes, my commit message says just that, but I can't remember why. I think that it had to do with the visualization of found indications when following the link from the build action. In this implementation, we aren't changing any visualization, just saving the line number for later use, so I don't think it will have much impact.